src/zarr/api/synchronous.py

```diff
@@ -895,6 +902,212 @@ def create_array(
                 storage_options=storage_options,
                 overwrite=overwrite,
                 config=config,
+                write_data=write_data,
+            )
+        )
+    )
+
+
+def from_array(
+    data: Array | npt.ArrayLike,
+    store: str | StoreLike,
+    write_data: bool = True,
+    *,
+    name: str | None = None,
+    chunks: Literal["auto", "keep"] | ChunkCoords = "keep",
+    shards: ShardsLike | None = None,
```

Comment:
```suggestion
    shards: ShardsLike | None = None,
```

should this also be `"keep"`?